runtime/onert/core/src/odc/QuantizeManager.test.cc

```diff
@@ -32,5 +32,6 @@ TEST(odc_QuantizeManager, neg_invalid_model_path)
 {
   QuantizeManager manager("invalid_model_path.circle");
   manager.exportModelPath("export_model_path.circle");
+  manager.quantizeType(ODC_QTYPE_WO_I8_SYM);
```

Comment:
Without this line, not-supported qtype error will be thrown before invalid model path.